library/common/config/config.cc

```diff
@@ -511,22 +595,7 @@ stats_sinks: *stats_sinks
   metadata: *metadata
 layered_runtime:
   layers:
-    - name: static_layer_0
-      static_layer:
-        envoy:
-          # This disables envoy bug stats, which are filtered out of our stats inclusion list anyway
-          # Global stats do not play well with engines with limited lifetimes
-          disallow_global_stats: true
-          reloadable_features:
-            allow_multiple_dns_addresses: *dns_multiple_addresses
-            always_use_v6: *force_ipv6
-            http2_delay_keepalive_timeout: *h2_delay_keepalive_timeout
-            skip_dns_lookup_for_proxied_requests: *skip_dns_lookup_for_proxied_requests
-)"
-// Needed due to warning in
-// https://github.com/envoyproxy/envoy/blob/6eb7e642d33f5a55b63c367188f09819925fca34/source/server/server.cc#L546
-R"(
-        overload:
-          global_downstream_max_connections: 0xffffffff # uint32 max
+#{custom_layers}
+#{custom_admin}
```

Comment:
don't the xDS clusters need to be added here as well through the `#{custom_clusters}` macro (or something similar)?

Reply:
I _think_ you just misread this one, I do add the custom clusters above (line 582)